Description: Fix XPCOM ABI name on ppc64el and ppc64
Author: Willi Mann <willi@debian.org>
--- enigmail-1.7.2.orig/config/getOsTarget.pl
+++ enigmail-1.7.2/config/getOsTarget.pl
@@ -76,6 +76,9 @@ if ($srcCpu =~ /x86[_-]64/i) {
 elsif ($srcCpu =~ /i[3456]86/i) {
   $targetCpu = "x86";
 }
+elsif ($srcCpu =~ /p(?:ower)?pc64/i) {
+  $targetCpu = "ppc64";
+}
 elsif ($srcCpu =~ /ppc/i) {
   $targetCpu = "ppc";
 }
